src/Property.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include <string>
+
+class Property
+{
+    std::string name_;
+    int purchasePrice_;
+public:
+    Property() = delete;
+    Property(const Property &) = default;
```

Comment:
I think we should not be able to copy a property card. If someone has one property card it should not be copyable, because then there is a possibility to have two owners.

Reply:
Good point, but I not sure about move constructor. Is it should be mark as delete?